src/zope/app/applicationcontrol/zopeversion.py

```diff
@@ -52,60 +44,13 @@ def getZopeVersion(self):
 
         self.result = "Development/Unknown"
 
-        # is this a SVN checkout?
-        svndir = os.path.join(self.path, ".svn")
-        if os.path.isdir(svndir):
-            self.__setSVNVersion(svndir)
-        else:
-            # try to get official Zope release information
-            versionfile = os.path.join(self.path, "version.txt")
-            if os.path.isfile(versionfile):
-                f = file(versionfile)
-                try:
-                    self.result = f.readline().strip() or self.result
-                finally:
-                    f.close()
-        return self.result
+        # try to get official Zope release information
+        versionfile = os.path.join(self.path, "version.txt")
+        if os.path.isfile(versionfile):
+            with open(versionfile) as f:
+                self.result = f.readline().strip() or self.result
```

Comment:
I think this bit can be removed -- a single `version.txt` to define the version of Zope 3 is a legacy from when we had a monolithic Subversion repository, before the eggsplosion.  It no longer makes sense.

But this leaves us with nothing here (or, rather, a constant 'Development/Unknown' "version").

I think what would make sense is to enumerate all the packages known to setuptools/pkg_resources and show their names and versions.  But that's a separate task, not a part of the Python 3 port PR.